modules/smil-workflowoperation/src/main/resources/OSGI-INF/operations/cut-marks-to-smil.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0"
+               name="org.opencastproject.workflow.handler.smil.CutMarksToSmilWorkflowOperationHandler"
+               immediate="true">
+  <implementation class="org.opencastproject.workflow.handler.smil.CutMarksToSmilWorkflowOperationHandler"/>
+  <property name="service.description" value="Cut Marks To Smil Operation Handler"/>
+  <property name="workflow.operation" value="cut-marks-to-smil"/>
```

Comment:
If anyone else is wondering: It would be nice to not specify this in XML but unfortunately, some methods are contained in the parent class which means we would probably need to change all the operation handlers at once.